build/props/common.props

```diff
@@ -45,7 +45,7 @@
 
             After this update:
             - Apple Silicon support is complete except for missing ANGLE binaries.
-            - Arm Linux support is complete except for missing DXVK, ANGLE, Vkd3d, and Vulkan Loader (missing on 32-bit only) binaries.
+            - Arm Linux support is complete except for missing DXVK, ANGLE, and Vulkan Loader (missing on 32-bit only) binaries.
```

Comment:
Note that Vulkan Loader doesn't actually fully support Arm32. So it's not just a shortcoming on Silk's end.

Reply:
Yep, was careful with the wording here i.e. saying "missing binaries" instead of "unsupported"